mybatis-plus/src/main/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+            </pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+    <logger name="com.baeldung.mybatisplus" level="DEBUG" additivity="false">
+        <appender-ref ref="STDOUT"/>
+    </logger>
+
+</configuration>
```

Comment:
I think we have a default logback configuration defined in the parent module, is there any reason for a custom logback.xml here?

Reply:
I want to show the DEBUG logs run by mybatis-plus classes. So, I have to add this logger `<logger name="com.baeldung.mybatisplus" level="DEBUG" additivity="false">`